src/features/identityCheck/atoms/ModalContent.tsx

```diff
@@ -0,0 +1,12 @@
+import styled from 'styled-components/native'
+
+import { getSpacing } from 'ui/theme'
+
+export const ModalContent = styled.View<{ centered?: boolean }>(({ centered = false }) => ({
+  width: '100%',
+  height: '100%',
+  maxWidth: getSpacing(125),
+  ...(centered && {
+    alignItems: 'center',
```

Comment:
```suggestion
    alignItems:  centered ? 'center' : undefined / 'flex-start',
```
? 

Pas fan de la syntaxe en destructuring `  ...(centered && {`